atom/browser/api/atom_api_web_contents.cc

```diff
@@ -406,6 +406,9 @@ void WebContents::InitWithSessionAndOptions(v8::Isolate* isolate,
 
   Init(isolate);
   AttachAsUserData(web_contents);
+
+  if (IsGuest())
+    Emit("did-create-webview");
```

Comment:
Do we still need this? We have `web-contents-created` that is emitted for each `webContents` and `webContents.getType()` to know which type was created.